app/view_models/auction_list_item.rb

```diff
@@ -38,6 +38,10 @@ def eligibility_label
     end
   end
 
+  def skills
+    auction.skills.map(&:name).join(', ')
```

Comment:
Should these be sorted (we could put a default order on the association even if you'd prefer)?